backend/store/migration/dev/LATEST.sql

```diff
@@ -374,6 +374,59 @@ UPDATE
     ON db FOR EACH ROW
 EXECUTE FUNCTION trigger_update_updated_ts();
 
+-- instance change history records the changes an instance and its databases.
+CREATE TABLE instance_change_history (
+    id BIGSERIAL PRIMARY KEY,
+    row_status row_status NOT NULL DEFAULT 'NORMAL',
+    creator_id INTEGER NOT NULL REFERENCES principal (id),
+    created_ts BIGINT NOT NULL DEFAULT extract(epoch from now()),
+    updater_id INTEGER NOT NULL REFERENCES principal (id),
+    updated_ts BIGINT NOT NULL DEFAULT extract(epoch from now()),
+    instance_id INTEGER NOT NULL REFERENCES instance (id),
+    -- NULL means an instance-level change.
+    database_id INTEGER REFERENCES db (id),
+    -- issue_id is nullable because this field is backfilled and may not be present.
+    issue_id INTEGER REFERENCES issue (id),
```

Comment:
ERROR: relation "issue" does not exist (SQLSTATE 42P01)

This needs to be put after the issue table